src/frontend/packages/core/src/shared/components/page-header/page-header.component.html

```diff
@@ -37,6 +37,24 @@
           ">
                 |
               </span>
+              <button mat-icon-button [matMenuTriggerFor]="token">
+                <mat-icon class="page-header__credential">vpn_key</mat-icon>
+              </button>
+              <mat-menu #token="matMenu" [overlapTrigger]="false" class="page-header__menu">
+                <div class="page-header__token">
+                  <h3 class="page-header__token-title">UAA Tokens</h3>
+                  <div (click)="copy(authToken$)" matTooltip="Click to copy">
+                    <h4>Auth Token</h4>
+                    <div class="page-header__token-text" hidden>{{ authToken$ | async }}</div>
+                  </div>
+                  <div (click)="copy(refreshToken$)" matTooltip="Click to copy">
+                    <h4>Refresh Token</h4>
+                    <div class="page-header__token-text" hidden>{{ refreshToken$ | async }}</div>
+                  </div>
+                  <h4>Token Expiry</h4>
+                  <div class="page-header__token-text">{{ tokenExpiry$ | async }}</div>
+                </div>
+              </mat-menu>
```

Comment:
<img width="296" height="310" alt="image" src="https://github.com/user-attachments/assets/57ed9b3d-21d7-4893-b5ef-b6a73c2e2dc2" />

```suggestion
              <mat-menu #token="matMenu" [overlapTrigger]="false" class="page-header__menu">
                <div class="page-header__token">
                  <h3 class="page-header__token-title">UAA Tokens</h3>
                  <button mat-stroked-button (click)="copy(authToken$)" matTooltip="Click to copy auth token">
                    <span>Copy Auth Token</span>
                    <div class="page-header__token-text" hidden>{{ authToken$ | async }}</div>
                  </button>
                  <button mat-stroked-button (click)="copy(refreshToken$)" matTooltip="Click to copy refresh token">
                    <span>Copy Refresh Token</span>
                    <div class="page-header__token-text" hidden>{{ refreshToken$ | async }}</div>
                  </button>
                  <h3>Token Expiry</h3>
                  <div class="page-header__token-text">{{ tokenExpiry$ | async }}</div>
                </div>
              </mat-menu>
```